src/pages/installation.md

```diff
@@ -3,58 +3,67 @@ layout: ~/layouts/Main.astro
 title: Installation
 ---
 
-There are a few different ways to install
+There are a few ways to getting started with Astro. Outlined below are instructions on how to go about installing Astro either manually or by our Astro Installer.
```

Comment:
Would "to get started" not sound better in this sentence?

Reply:
it is, ill make that ammendment